modules/acquia_cms_headless/tests/src/Functional/DashboardApiUsersTest.php

```diff
@@ -60,7 +60,7 @@ public function testSection(): void {
 
     // Test table body exist and has data in same order.
     $this->assertEquals('Headless', $this->getTableBodyColumn(0)->getText());
-    $this->assertEquals('Headless Role', $this->getTableBodyColumn(1)->getText());
+    $this->assertEquals('Headless Administrator', $this->getTableBodyColumn(1)->getText());
```

Comment:
@chandan-singh7929 Rebase from feature/headless_test_coverage and revert this change
This change is already pushed into PR: https://github.com/acquia/acquia_cms/pull/1431/files#diff-5570e81b9d7033df76b458d3155cf5546824e4b1f5be5b9dd984006cdd5b2da7R63

Reply:
Done